runbotics/runbotics-orchestrator-ui/src/views/bot/BotDetailsView/Header.tsx

```diff
@@ -17,14 +17,15 @@ const StyledGrid = styled(Grid)(() => ({
 }));
 
 const Header: FC = () => {
-    const { id } = useParams<BotParams>();
+    const router = useRouter();
+    const { id } = router.query;
```

Comment:
can we pass types for query params? then there will be no need to assert the type of queries

Reply:
I havent found a way to do that